src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,48 @@
+package parser;
+
+import command.*;
```

Comment:
I didn't like the use of * in import because it violates the coding standard. It might be an improvement to list all imports explicitly